docs/distributed/README.md

```diff
@@ -66,7 +68,12 @@ export MINIO_SECRET_KEY=<SECRET_KEY>
 minio server http://host{1...32}/export{1...32} http://host{33...64}/export{1...32}
 ```
 
-Now the server has expanded storage of *1024* more disks in total of *2048* disks, new object upload requests automatically start using the least used cluster. This expansion strategy works endlessly, so you can perpetually expand your clusters as needed.  When you restart, it is immediate and non-disruptive to the applications. Each group of servers in the command-line is called a zone. There are 2 zones in this example. Objects are placed in zones based on which zone has the *fmost free* space. Within each zone, the location of the erasure-set of drives is determined based on a deterministic hashing algorithm.
+Now the server has expanded storage of *1024* more disks in total of *2048* disks, new object upload requests automatically start using the least used cluster. This expansion strategy works endlessly, so you can perpetually expand your clusters as needed.  When you restart, it is immediate and non-disruptive to the applications. Each group of servers in the command-line is called a zone. There are 2 zones in this example. Objects are placed in zones based on which zone has the *most free* space. Within each zone, the location of the erasure-set of drives is determined based on a deterministic hashing algorithm.
```

Comment:
```suggestion
Now the server has expanded storage of *1024* more disks in total of *2048* disks, new object upload requests automatically start using the new cluster. This expansion strategy works endlessly, so you can perpetually expand your clusters as needed. When you restart, it is immediate and non-disruptive to the applications. 

Each group of servers in the command-line is called a zone. There are 2 zones in this example. New objects are distributed in zones based on available free space. Within each zone, the location of the erasure-set of drives is determined based on a deterministic hashing algorithm.
```

* Remove the "least used" since it is unclear how it relates to the zones.
* "most free" indicates only the one with most free space is selected. That is not the case IIRC.

Reply:
@klauspost 

```go
func getAvailableZoneIdx(ctx context.Context) int {
        zones := z.getZonesAvailableSpace(ctx)
        total := zones.TotalAvailable()
        // choose when we reach this many
        choose := rand.Uint64() % total
        atTotal := uint64(0)
        for _, zone := range zones {
                atTotal += zone.Available
                if atTotal > choose && zone.Available > 0 {
                        return zone.Index
                }
        }
        // Should not happen, but print values just in case.
        panic(fmt.Errorf("reached end of zones (total: %v, atTotal: %v, choose: %v)", total, atTotal, choose))
}
```

This code gives you the most available space zone, that's why `most free` perhaps mentioned here.

Reply:
@harshavardhana No, it gives the most probability for the zone with the most free space. It does not always choose the one with the most free space which 'most free' implies.

Reply:
But I see your point. Let me see if I can make it clearer.

Reply:
> Objects are placed in zones based on which zone has the *most free* space.

To me that indicates the zone with the most free space is chosen.

Reply:
In essence it is still free space based, while the wording may not be technically correct. I guess essence is still the same.

Reply:
How about: "New objects are placed in zones in proportion to the amount of free space in each zone"?